cmd/manager/main.go

```diff
@@ -228,6 +264,9 @@ func main() {
 			KlusterletWorkLister:     listersworkv1.NewManifestWorkLister(klusterletWorksInformer.GetIndexer()),
 			HostedWorkInformer:       hostedWorksInformer,
 			HostedWorkLister:         listersworkv1.NewManifestWorkLister(hostedWorksInformer.GetIndexer()),
+			KlusterletConfigInformer: klusterletconfigInformer,
```

Comment:
it is better to check if infomer has synced before starting the reconcile.

Reply:
Agree, added `WaitForCacheSync` for new informers.